tests/integration/kubernetes/gha-run.sh

```diff
@@ -175,7 +175,7 @@ function deploy_kata() {
 
 	ANNOTATIONS="default_vcpus"
 	if [[ "${KATA_HOST_OS}" = "cbl-mariner" ]]; then
-		ANNOTATIONS="image kernel default_vcpus disable_image_nvdimm"
+		ANNOTATIONS="image kernel default_vcpus disable_image_nvdimm cc_init_data"
```

Comment:
@Redent0r , I suggest splitting this commit into 3:
1. Delete init data annotation from the Host.
2. Add CLH support for init data.
3. Allow cbl-mariner to test using init data annotations - by modifying this ANNOTATIONS value.

Reply:
updated